### PR TITLE
Nc news 8

### PR DESCRIPTION
**Pull request by IanTibble1**

### CODE REVIEW

**Comment by IanTibble1** [comment]
Had some merging issues with this one which is why there are some odd commits

File: __tests__/app.test.js [hunk 1]
```diff
@@ -194,117 +184,131 @@ describe("app()", () => {
...
-    test("GET 400: /api/articles/:article_id should return a 400 message if id wrong datatype", () => {
+    test("GET 400: /api/articles/:article_id should return a 400 message if bad request made  ", () => {
       return request(app)
         .get("/api/articles/hello/comments")
         .expect(400)
         .then(({ body }) => {
           expect(body.msg).toBe("bad request");
         });
     });
+
```
Comment (Hard-Luck) [inline __tests__/app.test.js]: This block should be removed

Comment (Hard-Luck) [reply __tests__/app.test.js]: Unless you meant for that expect 200 not to be commented out

File: __tests__/app.test.js [hunk 2]
```diff
@@ -194,117 +184,131 @@ describe("app()", () => {
...
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: "2020-07-09T20:11:00.000Z",
+            votes: 0,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          });
         });
     });
```
Comment (Hard-Luck) [inline __tests__/app.test.js:282]: A test for an incorect body would be good too for instance if the "inc_votes" key is missing or if the req body is {inc_votes: "not a number"}

**Review by Hard-Luck — APPROVED** [review]
Just a bit of tidying up required, make sure you remove commented code, then you can merge

**Pull request merged**